rust/Cargo.lock

```diff
@@ -37,6 +37,11 @@ version = "0.8.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
I'm super new to rust, but from the reading I've done it seems like we can exclude the cargo.lock file for this PR.